File: JavaSpringService/src/main/java/com/polyglott/JavaSpringService/Item.java
package com.polyglott.JavaSpringService;

public class Item {
    private String name;
    private int dmg;

   Item (){

   }

    Item(String name, int dmg){
        this.name = name;
        this.dmg = dmg;
    }

    public int getDmg() {
        return dmg;
    }

    public void setDmg(int dmg) {
        this.dmg = dmg;
    }

    public String getName() {

        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: pythonDjango/pythonDjango/pythonDjango/item.py
from django.db import models


class Item(models.Model):

#    id = models.IntegerField()
    name = models.CharField(max_length=50)
    dmg = models.IntegerField()

    def __str__(self):
        istring ="Name:" +self.name +" Dmg:" +str(self.dmg) +" Id:" + self.id *" "
        return istring


    def get_name(self):
        return self.name




